authorizations/authorization-ranger/src/test/java/org/apache/gravitino/authorization/ranger/integration/test/RangerITEnv.java

```diff
@@ -156,11 +161,30 @@ public static void init(String metalakeName, boolean allowAnyoneAccessHDFS) {
             rangerAuthHDFSPlugin.ownerMappingRule(),
             rangerAuthHDFSPlugin.policyResourceDefinesRule());
 
+    // Test to use UUID create authz plugin
+    new RangerAuthorizationHadoopSQLPlugin(
+        metalakeName,
+        ImmutableMap.of(
+            RangerAuthorizationProperties.RANGER_ADMIN_URL,
+            String.format(
+                "http://%s:%d",
+                containerSuite.getRangerContainer().getContainerIpAddress(),
+                RangerContainer.RANGER_SERVER_PORT),
+            RangerAuthorizationProperties.RANGER_AUTH_TYPE,
+            RangerContainer.authType,
+            RangerAuthorizationProperties.RANGER_USERNAME,
+            RangerContainer.rangerUserName,
+            RangerAuthorizationProperties.RANGER_PASSWORD,
+            RangerContainer.rangerPassword,
+            BaseAuthorization.UUID,
+            "123",
+            RangerAuthorizationProperties.RANGER_SERVICE_TYPE,
+            "HadoopSQL",
+            RangerAuthorizationProperties.RANGER_SERVICE_CREATE_IF_ABSENT,
+            "true"));
```

Comment:
I think it would be better to add a check to see if Ranger service success was created.